src/utilities/tooltip.ts

```diff
@@ -39,3 +43,24 @@ export function tooltip(node: Element, params: any = {}): any {
     update: (newParams: any) => tip.setProps({ content, ...newParams }),
   };
 }
+
+const disabled = {
```

Comment:
does it make sense to declare the type for `disabled` so that it's more immediately clear what it is?